pkg/controller/jaeger/consolelink_test.go

```diff
@@ -0,0 +1,201 @@
+package jaeger
+
+import (
+	"context"
+	"testing"
+
+	osconsolev1 "github.com/openshift/api/console/v1"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	v1 "github.com/jaegertracing/jaeger-operator/pkg/apis/jaegertracing/v1"
+	"github.com/jaegertracing/jaeger-operator/pkg/strategy"
+)
+
+func TestConsoleLinkCreate(t *testing.T) {
+	// prepare
+	nsn := types.NamespacedName{
+		Name: "TestCronJobsCreate",
```

Comment:
This name is wrong. Please, use `my-instance` for new tests, as having unique names isn't bringing much value (and causes copy/paste problems like the above ;) )

Reply:
ooops, you caught me.. I did a copy&paste..  :flushed: I'll fix this.